docker-compose.dev.yml

```diff
@@ -27,7 +27,7 @@ services:
       context: ./server
       dockerfile: Dockerfile.dev
       args:
-        VERSION: ${LAST_COMMIT}
+        VERSION: ${COMMIT_SHA}
```

Comment:
Does this work as a long-term solution to the Docker Engine error?

Reply:
I think it might have too. I have a theory that a length limit has been applied to text fields, and the `LAST_COMMIT` variable is too long. `COMMIT_SHA` is shorter and should be unique.